packages/cli/src/commands/generate/auth/providers/auth0.js

```diff
@@ -4,8 +4,7 @@ export const config = {
   init: `const auth0 = new Auth0Client({
     domain: process.env.AUTH0_DOMAIN,
     client_id: process.env.AUTH0_CLIENT_ID,
-    redirect_uri: 'http://localhost:8910/',
-    cacheLocation: 'localstorage',
```

Comment:
I removed setting the cacheLocation to localstorage so that the Auth0Client uses the default "in memory" per Auth0's recommendations.

```
This can help to mitigate some of the effects of browser privacy technology that prevents access to the Auth0 session cookie by storing Access Tokens for longer.
```

See: https://auth0.com/docs/tokens/concepts/token-storage#browser-in-memory-scenarios

and

https://auth0.com/docs/libraries/auth0-spa-js#change-storage-options


```
**Storing tokens in browser local storage provides persistence across page refreshes and browser tabs. However, if an attacker can achieve running JavaScript in the SPA using a cross-site scripting (XSS) attack, they can retrieve the tokens stored in local storage. A vulnerability leading to a successful XSS attack can be either in the SPA source code or in any third-party JavaScript code (such as bootstrap, jQuery, or Google Analytics) included in the SPA.**
```

Note, there are some drawbacks:

```
The in-memory method for browser storage does not provide persistence across page refreshes and browser tabs.
```

Reply:
I think that we should provide a warning, and let the implementor decide the risk, since refreshing the browser would cause you to be unauthenticated this would be a bad experience for both the developer and the users.

Reply:
> since refreshing the browser would cause you to be unauthenticated this would be a bad experience

True.

* [ ] add a comment inline
* [ ] a note in the auth0 template generator

?

Reply:
I think we've started heading into out of scope territory for this PR, and I would love to merge it, but before I do, I want to make a few changes to the new structure that you've introduced! 

Could you give me commit access to your fork?

Reply:
@peterp yes, I completely agree. it's time to wrap up ... before this wanders too far afield. I'll grant access asap.